client/app/views/account.coffee

```diff
@@ -108,7 +129,116 @@ module.exports = class exports.AccountView extends BaseView
                         , 10000
                     else
                         showError 'account change password error'
+                        
+    # 2fa
+    on2faEnableSubmit: (event) ->
+        authType = @twoFactorsField.val()
+        hideFunc = null
+        showSuccess = (message) =>
+            @info2faAlert.html t message
+            @info2faAlert.fadeIn()
+            clearTimeout hideFunc
+            hideFunc = setTimeout =>
+                window.location.reload()
+            , 2000
+            
+        form =
+            authType: authType
+            otpKey: @getOtpKey()
+            hotpCounter: 0 if authType is 'hotp'
```

Comment:
Yes, as a new OTP key is generated at each submission (and overrides an old one if present). I think we must suppose each activation (or deactivation followed by activation) of OTP will need the user to reconfigure its 2FA device or app like the first time, for more safety.